sunkit_image/_dev/scm_version.py

```diff
@@ -1,14 +1,12 @@
 # Try to use setuptools_scm to get the current version; this is only used
 # in development installations from the git repository.
-from pathlib import Path
+import os.path
```

Comment:
This should be updated in the template to use Pathlib?

Reply:
yeah it does, will sort it